pkg/dataconn/client.go

```diff
@@ -217,10 +217,26 @@ func (c *Client) handleResponse(resp *Message) {
 
 		journal.RemovePendingOp(req.ID, true)
 		delete(c.messages, resp.Seq)
+
+		if req.Type == TypeRead || req.Type == TypeWrite {
+			// We received a response to an I/O operation
+			// Stop the timer.
+			c.timer.Stop()
+			c.iops--
+
+			// If there are other reads and writes in
+			// progress, wait opTimeout seconds for any
+			// response to an I/O operation.
+			if c.iops > 0 {
+				c.timer = time.NewTimer(opTimeout)
```

Comment:
Can we reset the timer instead of recreating one? Saving GC efforts.

Reply:
I changed it to reset.